crates/libcontainer/src/process/fork.rs

```diff
@@ -164,12 +164,12 @@ fn clone(cb: CloneCb, flags: u64, exit_signal: Option<u64>) -> Result<Pid, Clone
     // do not use MAP_GROWSDOWN since it is not well supported.
     // Ref: https://man7.org/linux/man-pages/man2/mmap.2.html
     let child_stack = unsafe {
-        mman::mmap(
+        mman::mmap::<File>(
```

Comment:
May I ask you to write a comment that `File` doesn't have any meaning because we won't use it.

Reply:
There's no specific reason. it's related to the change in the function signature of `nix::sys::mman::mmap()`. The current signature of the `mmap()` function is:
```rs
pub unsafe fn mmap<F: AsFd>(
    addr: Option<NonZeroUsize>, 
    length: NonZeroUsize, 
    prot: ProtFlags, 
    flags: MapFlags, 
    f: Option<F>, 
    offset: off_t) -> Result<*mut c_void>
```
If you remove `::<File>`, it will result in a compilation error:
> type annotations needed; cannot infer type of the type parameter `F` declared on the function `mmap`

Is there any other way to avoid the compilation error?

Reply:
I understood it, but it's unclear to the reader of this code. May I ask you to write a comment about it?